x/profile/types/message_create_coordinator_test.go

```diff
@@ -1,29 +1,30 @@
-package types
+package types_test
```

Comment:
Why change the package name? Import cycle?

Reply:
Yes. The `sample` package need the types but the tests need the samples

Reply:
And also I believe this is a good practice to not expose the test functions in the main package

Reply:
I like it for some situations, but not works if we want to test a non-exported method (lowercase)

Reply:
usually, I like to use tests with another package name for BlackBox tests, to test only the exported methods, and test with the same package name for white-box tests

Reply:
Yeah, sometime it can be useful. I don't see the necessity for `types` so far. For example, I uppercased the `ValidateValidators` and `ValidateCoordinators` for genesis validation. I don't think this is problematic

Reply:
This is particular to types. I don't think we will need to do this for `keeper`